sematic/cli/settings.py

```diff
@@ -35,11 +35,22 @@ def show_settings_cli() -> None:
 @settings.command("set", short_help="Set a user settings value")
 @click.argument("var", type=click.STRING)
 @click.argument("value", type=click.STRING)
-def set_settings_cli(var: str, value: str) -> None:
+@click.option(
+    "-p",
+    "--plugin",
+    "plugin",
+    type=click.STRING,
+    help="Import path to a plugin to configure a setting for.",
+    default=None,
+)
+def set_settings_cli(var: str, value: str, plugin: Optional[str]) -> None:
     """
     Set a setting value.
     """
-    _set_plugin_settings_cli(var, value, UserSettings)
+    plugin_class: Type[AbstractPlugin] = UserSettings
+    if plugin is not None:
+        plugin_class = import_plugin(plugin)
```

Comment:
Please add a user-friendly message that the plugin was not found on `ImportError`.

Reply:
Current error is:
```
sematic.utils.exceptions.MissingPluginError: Unable to find plug-in sematic.plugins.storage.s3_storage.S3StoZrage. Module or class is missing.
```
I think that's pretty good!